docs/UserGuide.md

```diff
@@ -3,12 +3,22 @@ layout: page
 title: User Guide
 ---
 
+## Introduction
+
+![App Logo](../src/main/resources/images/guest_book.png)
+
 GuestBook is a **desktop app for managing guests in a hotel,
 optimized for use via a Command Line Interface** (CLI)
 while still having the benefits of a Graphical User Interface (GUI).
 If you can type fast, GuestBook can get your
 guest management tasks done faster than traditional GUI apps.
 
+You can use GuestBook to track guest and room details such as emails, phone numbers, room numbers, dates of stay,
```

Comment:
Should this be "track guests" instead of "guest"?

Reply:
Modified